packages/purpose-process/test/activatePurposeVersion.test.ts

```diff
@@ -703,6 +708,67 @@ describe("activatePurposeVersion", () => {
     }).rejects.toThrowError(organizationNotAllowed(anotherTenant.id));
   });
 
+  it("should throw organizationNotAllowed if the caller is the producer but the purpose e-service has an active delegation", async () => {
+    await addOnePurpose(mockPurpose);
+    await writeInReadmodel(toReadModelEService(mockEService), eservices);
+    await writeInReadmodel(toReadModelAgreement(mockAgreement), agreements);
+    await writeInReadmodel(toReadModelTenant(mockConsumer), tenants);
+    await writeInReadmodel(toReadModelTenant(mockProducer), tenants);
```

Comment:
Can we use `addOneEservice` and so on in place of directly `writeInReadmodel`? I know this is how it is written the rest of the test but maybe we could change also the rest of the test.